client/src/index.jsx

```diff
@@ -16,7 +16,7 @@ ReactDOM.render(
   <React.StrictMode>
     <Router>
       <Switch>
-        <Route path='/recipe/:recipeId'>
+        <Route path='/recipeId'>
```

Comment:
This change will eventually need to be reverted to allow for recipeId as a parameter, rather than the static path